_guides/guide.md

```diff
@@ -38,15 +47,10 @@ Then delete the copied exercise repositories from a previous workshop.
 - [{{ site.centralized_workflow_exercise_url }}]({{ site.centralized_workflow_exercise_url }})
 - [{{ site.forking_workflow_exercise_url }}]({{ site.forking_workflow_exercise_url }})
 
-Now use the [GitHub importer](https://github.com/new/import) to create new copies:
-
-Centralized exercise:
-- Your old repository’s clone URL: [https://github.com/coderefinery/template-centralized-workflow-exercise](https://github.com/coderefinery/template-centralized-workflow-exercise)
-- Your new repository details: [{{ site.centralized_workflow_exercise_url }}]({{ site.centralized_workflow_exercise_url }})
-
-Forking exercise:
-- Your old repository’s clone URL: [https://github.com/coderefinery/template-forking-workflow-exercise](https://github.com/coderefinery/template-forking-workflow-exercise)
-- Your new repository details: [{{ site.forking_workflow_exercise_url }}]({{ site.forking_workflow_exercise_url }})
+Now create the two exercises by [importing](https://help.github.com/en/articles/creating-a-repository-from-a-template)
+these two templates:
+- [https://github.com/coderefinery/template-centralized-workflow-exercise](https://github.com/coderefinery/template-centralized-workflow-exercise)
+- [https://github.com/coderefinery/template-forking-workflow-exercise](https://github.com/coderefinery/template-forking-workflow-exercise)
```

Comment:
If we mostly use this for CodeRefinery, should we make clear what the default CodeRefinery names are so it matches the default CodeRefinery lesson?

Reply:
Oh yes. This somehow fell of the desk during my refactoring. I will add a sentence on that.

Reply:
I have added a word but think that it is clear from the guide now which repositories to create. The names will automatically change according to the configuration in `_config.yml`.